ci/playbooks/kuttl/run-kuttl-tests.yml

```
@@ -61,3 +61,30 @@
     {{ item }} > {{ cifmw_artifacts_basedir }}/logs/cmd_after_{{ operator }}_kuttl.log
   loop: "{{ commands_after_kuttl_run }}"
   ignore_errors: true
+
+# Same of the xml files may not be the JUnitXML results, but getting the exact file
+# name (parsing the KUTTL configuration) may be too cumbersome.
+- name: 'Find the generated JUnitXML files'
+  ansible.builtin.find:
+    paths: "{{ cifmw_installyamls_repos }}"
+    file_type: file
+    patterns: '*.xml'
+  register: "_cifmw_kuttl_xml_files"
+
+- name: 'Copy the log results of {{ operator }}'
```

nit: For task names with template vars this project normally uses double quotes. 
```suggestion
- name: "Copy the log results of {{ operator }}"
```